common/persistence/sql/dbal/class.Driver.php

```diff
@@ -265,4 +267,24 @@ public function transactional(Closure $func)
     {
         return $this->connection->transactional($func);
     }
+
+    /**
+     * add quotes to column names in column associated array
+     */
+    private function clearQuoteColumns(array $columnsAssociatedArray): array
```

Comment:
Just by looking at the name of the method, I'm getting confused about its purpose – does it clear the quotes, does it quote the columns, does it do both? :)

WDYT of renaming it to `quoteColumnsMap()`?

Reply:
this name goes from initial `$cleanColumns` variable name
agree probably better rename it